superset-frontend/plugins/plugin-chart-echarts/src/Bubble/controlPanel.tsx

```diff
@@ -138,6 +138,7 @@ const config: ControlPanelConfig = {
               choices: [
                 [0, '0°'],
                 [45, '45°'],
+                [90, '90°'],
```

Comment:
@michael-s-molina is the last choice assigned the default?

Reply:
Sorry I misread your PR description. I initially read it as you were setting `90°` as the default as opposed to adding it as one of the default options.